### PR TITLE
Adding max height of the app, better dictionary support, removing scollbar

### PR DESCRIPTION
**Pull request by SupremeSalvatore**
Adding max height of the app, better dictionary support, removing scollbar

### CODE REVIEW

File: assets/src/libs/ZDClient.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ const ZDClient = {
    * @param {Int} newHeight
    */
   resizeFrame(appHeight) {
-    CLIENT.invoke('resize', {width: '100%', height: `${appHeight}px`});
+    const maxAppHeight=appHeight>500 ? 500 : appHeight;
```
Comment (fabioguarrasi) [inline assets/src/libs/ZDClient.js]: can you please add spaces like
```javascript 
const maxAppHeight = appHeight > 500 ? 500 : appHeight;
```

File: assets/src/i18n/index.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
 import dictionary from './dictionary.js';
 
+function getObjProperty(object, propertyName) {
```
Comment (fabioguarrasi) [inline assets/src/i18n/index.js:6]: could you please add a method description so that everyone understands how this method works? Thanks :)

File: assets/style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+html{
+  overflow-y:auto;
```
Comment (fabioguarrasi) [inline assets/style.css:2]: why is this needed?

**Review by fabioguarrasi — CHANGES_REQUESTED** [review]

File: assets/style.css (hunk 3 above)
Comment (SupremeSalvatore) [reply assets/style.css:2]: Hiding useless scrollbar on right, and when it goes above our new max height it will appear

Comment (fabioguarrasi) [reply assets/style.css:2]: can you please add a space after `:`? thanks

File: assets/src/libs/ZDClient.js [hunk 4]
```diff
@@ -22,7 +22,8 @@ const ZDClient = {
    * Set getters for privite objects
    */
   app: {
-    get settings() { return APP_SETTINGS; }
+    get settings() { return APP_SETTINGS; },
+    get client() { return CLIENT; }
```
Comment (fabioguarrasi) [inline assets/src/libs/ZDClient.js:26]: why should we expose the client outside?

Comment (SupremeSalvatore) [reply assets/src/libs/ZDClient.js:26]: we should not, i actually never used it outside of that so i will be removing this, hopefully i know where is this pull request :D
